Himanshu_Sharma/project3_company/project3_company/Controllers/FormController.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using project3_company.CustomModels.Models;
+using project3_company.CustomModels.Repositories;
+
+namespace project3_company.Controllers
+{
+    [ApiController]
+    public class FormController : ControllerBase
+    {
+        [Authorize]
+        [HttpPost("api/form/create_employee")]
```

Comment:
Define the Route Parameter on top so you don't have to use "api/form" repeatedly on every function

Reply:
Fixed!